Makefile

```diff
@@ -45,6 +45,14 @@ markdown-link-check:
 shellcheck:
 	./ci/shellcheck.sh
 
+.PHONY: install-pre-commit
+install-pre-commit:
+	python3 -m pip install pre-commit
```

Comment:
What about target to install python3 :hand_over_mouth:

Reply:
We have to make some assumptions 😄 Our Makefile currentkly assumes that you have both Python 3 and Node.js installed.